carmajava/signal_plugin/src/main/java/gov/dot/fhwa/saxton/carma/signal_plugin/TrafficSignalPlugin.java

```diff
@@ -381,6 +400,8 @@ static protected SpatMessage convertSpatMessage(IntersectionData data) {
                         //log.warn("Unsupported signal phase: " + phase);
                         break;
                 }
+                // TODO Uncomment to disable lights
```

Comment:
Is this (and line 382 above) something likely to be useful again? Maybe worth adding an `ignore_lights` param?